samples/matter/window_covering/README.rst

```diff
@@ -38,6 +38,22 @@ The following movement modes are available:
 
 See `User interface`_ for information about how to switch the movement modes.
 
+Synchronized Sleepy End Device activity
+=======================================
+
+Because it is working as a Thread Synchronized Sleepy End Device, the window covering device based on this sample is synchronized with its parent router and uses the radio only at scheduled intervals.
```

Comment:
```suggestion
Because it is working as a Thread Synchronized Sleepy End Device, the window covering device based on this sample is synchronized with its parent router and uses the radio only at scheduled intervals, by utilizing Thread 1.2 Coordinated Sampled Listening (CSL) feature.
```